integrationTests/cypress/integration/apd/01-apd-basics.js

```diff
@@ -2,4 +2,217 @@
 
 // Tests performing basic APD tasks
 
-describe('APD Basics', () => {});
+/* eslint-disable no-return-assign */
+/* eslint-disable prefer-arrow-callback */
+
+describe('filling out APD overview section', function () {
+  const page_titles = ['APD Overview', 'Key State Personnel', 'Results of Previous Activities', 'Activities', 'Activity Schedule Summary', 'Proposed Budget', 'Assurances and Compliance', 'Executive Summary', 'Export and Submit'];
+
+  before(() => {
+    cy.useStateStaff();
+    // Delete all existing APDs
+    cy.get('button').each($el => {
+      if ($el.text().includes('Delete')) {
+        cy.wrap($el).click();
+        cy.findAllByText('Delete APD').click();
+      }
+    });
+    cy.wait(1000); // Wait for saving
+  });
+
+  it('creates a new APD with current date', () => {
+    var options = { year: 'numeric', month: 'long', day: 'numeric' };
+    var today  = new Date();
+
+    cy.contains('Create new').click();
+
+    cy.get('.apd--title').contains("Created: " + today.toLocaleDateString("en-US", options));
+  });
+
+  it('confirms Continue buttons redirect to correct sections', () => {
+    cy.wrap(page_titles).each((index) => {
+      cy.get('.ds-h2')
+        .should('contain', index);
+
+      if(index != page_titles[page_titles.length - 1]) {
+        cy.get('#continue-button').click();
+      }
+    });
+  });
+
+  it('confirms Back buttons redirect to correct sections', () => {
+    const reverse_page_titles = page_titles.reverse()
+
+    cy.wrap(reverse_page_titles).each((index) => {
+      cy.get('.ds-h2')
+        .should('contain', index);
+
+      if(index != reverse_page_titles[reverse_page_titles.length - 1]) {
+        cy.get('#previous-button').click();
+      }
+    });
+  });
+
+  it('confirms side nav buttons redirect to correct sections', () => {
```

Comment:
Not critical but we could probably have an object of the nav items and loop over them. Similar to the command comment I had below. Maybe there's a way to do it with cypress fixtures or pages, maybe @thetif knows.

Reply:
I took both notes and applied them!